code_notebook(samples_images).md

```diff
@@ -0,0 +1,19 @@
+## NOTEBOOK (CODE IMAGES)
```

Comment:
What does this file do exactly?

Reply:
i removed this and added jupyter notebook to markdown file

Reply:
No need to convert the ipynb to md, you can directly commit the notebook and delete all the code images

Reply:
Also you need to change the deployed app on heroku and give everyone the credits for it including WinterofCode, @paltannistha , @khanfarhan10 , @Vamsi995 , @Ankitasareen and yourself @Dineshkumar536 . Also you need to redeploy it as your fellowmates are regularly making changes to the repository.

However, you can keep the developer credits at the bottom of the page as you have made the app yourself.